services/movie.py

```diff
@@ -15,18 +16,22 @@
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
 
 
+@transaction.atomic
 def create_movie(
     movie_title: str,
     movie_description: str,
     genres_ids: list = None,
-    actors_ids: list = None,
+    actors_ids: list = None
```

Comment:
For `genres_ids` and `actors_ids`, consider using `Optional[list[int]]` to indicate that these can be `None`. This will improve type safety and clarity.